### PR TITLE
S2 splash screen

### PR DESCRIPTION
**Pull request by dmayarace**

### CODE REVIEW

**Comment by mclack** [comment]
Looks great! I hereby bestow upon thee my seal of approval

**Review by mclack — APPROVED** [review]

**Comment by kimberleyworld** [comment]
I love it, just the mentioned margin thing and it's good to go!

**Comment by dmayarace** [comment]
Margin changes

File: index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Lynx Sweeper</title>
+    <link rel="stylesheet" href="styles.css" type="text/css">
```
Comment (charliecog) [inline index.html:6]: needs normalise.css

File: index.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Lynx Sweeper</title>
```
Comment (charliecog) [inline index.html:5]: needs the mobile device meta tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    <link rel="stylesheet" href="styles.css" type="text/css">
+</head>
+<body>
+<header>
+    <h1>Lynx Sweeper</h1>
+    <div class="instructions">
+        <h2>Instructions:</h2>
+        <p> Lynx have gone missing and it's your job to find them.</p>
```
Comment (charliecog) [inline index.html:13]: maybe gramatically it should read "Some Lynx ......"

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    align-items: center;
+    flex-flow: column;
+}
+
+h1 {
+    color: #355f4d;
+    font-size: 5rem;
+    margin: 60px 40px;
+}
+.instructions{
```
Comment (charliecog) [inline styles.css:24]: needs a line break above

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    background-color: rgba(224,216,222, 0.5);
+    padding: 30px 40px;
+    border-radius: 5px;
+    margin: 20px 40px;
+}
+h2 {
+    font-size: 2rem;
+}
+
+p{
```
Comment (charliecog) [inline styles.css:34]: needs a space between p and the {

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,36 @@
+*{
```
Comment (charliecog) [inline styles.css:1]: needs a space between * and {

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+    box-sizing: border-box;
+
```
Comment (charliecog) [inline styles.css:3]: remove extra line break

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+}
+body {
```
Comment (charliecog) [inline styles.css:5]: needs a line break above

**Review by charliecog — APPROVED** [review]

**Pull request merged**